plugins/parcelport/verbs/scheduler.hpp

```diff
@@ -0,0 +1,132 @@
+//  Copyright (c) 2015-2016 John Biddiscombe
+//
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
```

Comment:
Yes, this scheduler is only enabled by cmake under user control and comes with a warning (see cmakelists). Work on custom schedulers is ongoing and I'd like to leave this code in place until I'm ready to replace it. Please feel free to create an issue to address custom parcelport exectutors as a placeholder to get rid of this.